aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/utils/DockerUtils.java

```diff
@@ -96,4 +105,70 @@ public String getContainerId() throws IOException {
 
         return null;
     }
+
+    /**
+     * Search a file path for the Docker daemon config file, and if present parse that file for the AWS Log group.
+     *
+     * Daemon config file: https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file
+     * awslogs driver format: https://docs.docker.com/config/containers/logging/awslogs/
+     *
+     * @param filePath file system path to daemon configuration file
+     * @param fs file system to search for config file
+     * @return an AWSLogReference for the discovered log group, or null if the config file is not present or does not
+     *         contain an awslogs driver
+     */
+    @Nullable
+    public static AWSLogReference getAwsLogReference(String filePath, FileSystem fs) {
```

Comment:
There doesn't seem to be a need to put this here instead of private where it's used